internal/rules/policies/cloud/policies/aws/rds/enable_deletion_protection.rego

```diff
@@ -0,0 +1,25 @@
+# METADATA
```

Comment:
This is the rego version of the cloudsploit plugin. The metadata can mostly be obtained from the original cloudsploit file.